### ISSUE TITLE
App from app template complains of mismatched ember-data and compiled templates

### ISSUE THREAD

**Issue by IamNaN** [emberjs/ember-rails#465]
Trying to work through getting an ember-rails app working from scratch. Following the readme I run:

    rails new myapp -m http://emberjs.com/edge_template.rb
    cd myapp
    rake db:create
    rails generate ember:install
    rake tmp:clear # to be thorough
    rails s

Since this is straight out-of-the-box generated app at this point, I'd expect to see app/assets/javascript/template/index.js.handlebars when I visit localhost:3000. However, it doesn't and the following javascript console has this error:

![screen shot 2015-03-31 at 1 02 37 pm](https://cloud.githubusercontent.com/assets/329533/6931243/8c3b15c8-d7a6-11e4-9b5c-ec7a9e25f989.png)

I'm pretty sure 1.11.0 is higher than 1.8.0.

Also, the last error regarding mismatched compiled templates seems separate.

**Comment by tricknotes** [comment]
Thanks @IamNaN .
This issue has some causes:

* Ember Data couldn't check Ember version with some suffix correctly. -> https://github.com/emberjs/data/pull/2965
* Ember Rails could't precompile templates with vendor assets. (It will keep tracking on this issue)

For temporary workaround, you can use release build of Ember.
```
$ curl http://builds.emberjs.com/tags/v1.11.0/ember.debug.js > vendor/assets/ember/development/ember.js
$ curl http://builds.emberjs.com/tags/v1.11.0/ember.prod.js > vendor/assets/ember/production/ember.js
```

**Comment by tricknotes** [comment]
> Ember Rails couldn't precompile templates with vendor assets.

This should be solved in barber. https://github.com/tchak/barber/issues/43
I'll fix it tonight.

**Comment by tricknotes** [comment]
> Ember Data couldn't check Ember version with some suffix correctly. -> emberjs/data#2965

This issue was fixed but not yet released.
To avoid, you can [use release channel of Ember](https://github.com/emberjs/ember-rails/issues/465#issuecomment-88342631) or canary channel Ember Data.

> Ember Rails couldn't precompile templates with vendor assets. (It will keep tracking on this issue)

This issues was fixed on [ember-rails@0.18.1](https://rubygems.org/gems/ember-rails/versions/0.18.1).

**Comment by IamNaN** [comment]
At the risk of adding length to this issue, I want to thank you for the updates, work-around info, and generally keeping the ember-rails project going so well like this. Cheers.
